comms/dht/src/crypt.rs

```diff
@@ -35,15 +35,16 @@ use tari_crypto::{
     keys::{DiffieHellmanSharedSecret, PublicKey},
     tari_utilities::{epoch_time::EpochTime, ByteArray},
 };
-use zeroize::{Zeroize, ZeroizeOnDrop};
+use zeroize::Zeroize;
 
 use crate::{
     envelope::{DhtMessageFlags, DhtMessageHeader, DhtMessageType, NodeDestination},
     outbound::DhtOutboundError,
     version::DhtProtocolVersion,
 };
 
-#[derive(Debug, Clone, Zeroize, ZeroizeOnDrop)]
+#[derive(Debug, Clone, Zeroize)]
+#[zeroize(drop)]
```

Comment:
In case if we use `chacha20poly1305` crate we should use `zeroize=1.4` to let cargo solve dependencies.